packages/InteractiveProfilingTool-Tests.package/IPTModelTestCase.class/instance/setUp.st

```diff
@@ -0,0 +1,5 @@
+running
+setUp
+
+	model := IPTModelMock new.
```

Comment:
Is there a particular reason for using `IPTModelMock` instead of `IPTModel`?

Reply:
Yes, `IPTModelMock` overwrites the `getMessageTally` method. The method implementation in the super class starts an endless loop that cannot be interrupted without user input. This class' implementation returns a mocked `MessageTally` object without starting an endless loop for testing purposes.